deploy/aws/README.md

```diff
@@ -67,14 +67,12 @@ please see the [official migration guide](https://helm.sh/docs/topics/v2_v3_migr
 
 > **NOTE**: Moving forward this chart will only be tested and maintained for Helm v3.
 
-Below are example instructions for installing Helm v2. 
+Below are example instructions for installing Helm v2.
 
 ```
-$ curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
-$ kubectl create serviceaccount -n kube-system tiller
-serviceaccount/tiller created
-$ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-$ helm init --service-account tiller --wait
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
```

Comment:
These are Helm 3 install instructions in the Helm 2 section, please keep the original Helm 2 instructions here: https://github.com/triton-inference-server/server/blob/main/deploy/gcp/README.md#helm-v2